bodhi/server/push.py

```diff
@@ -50,8 +50,28 @@ def update_sig_status(update):
                 click.echo('Build %s still unsigned' % build.nvr)
 
 
+def check_if_updates_and_builds_set(ctx, param, value):
```

Comment:
For new code, we can start using type annotations for types instead of docblocks - it'll make the conversion easier later. It's also OK to do this for consistency, if you prefer.